src/getThreadList.js

```diff
@@ -4,23 +4,32 @@ var utils = require("../utils");
 var log = require("npmlog");
 
 module.exports = function(defaultFuncs, api, ctx) {
-  return function getThreadList(start, end, callback) {
+  return function getThreadList(start, end, isPending, callback) {
     if(!callback && utils.getType(end) !== 'Number') {
       throw {error: "please pass an number as a second argument."};
     }
 
+    if(!callback && utils.getType(isPending) !== 'Boolean') {
+      throw {error: "please pass a Boolean as a third argument."};
+    }
+
     if(!callback) {
       throw {error: "getThreadList: need callback"};
     }
 
     if (end <= start) end = start + 20;
 
-    var form = {
-      'client' : 'mercury',
-      'inbox[offset]' : start,
-      'inbox[limit]' : end - start,
+    var messageBox = 'inbox';
+    if (isPending) {
+      messageBox = 'pending';
     };
```

Comment:
I usually feel like 
`var messageBox = pending ? "pending" : "inbox";`
is more explicit.
I would have also probably called it `pending` rather than `isPending` (and accept rather than isAccept) but I guess that's getting reeeaally nitpicky.